lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/takeaway.rb'
+class Order
+  attr_reader :items, :menu
+  def initialize(menu)
+    @items = {}
+    @menu = menu
+  end
+
+  def add_dish(dish, quantity = 1)
+    @items[dish] = 0 unless items[dish]
+    @items[dish] += quantity
+  end
+
+  def single_item_price(dish)
+    @menu.dishes[dish]
+  end
+
+  def order_price
+    total = 0
```

Comment:
Try looking at the .reduce method here, it would mean you don't need to have line 19 or 23